addons/website_event_track_quiz/static/src/js/event_quiz.js

```diff
@@ -132,29 +135,30 @@ var Quiz = publicWidget.Widget.extend({
         this.$('.o_quiz_js_quiz_question').each(function () {
             var $question = $(this);
             var questionId = $question.data('questionId');
-            var isCorrect = self.quiz.answers[questionId].is_correct;
+            var answer = self.quiz.answers[questionId];
             $question.find('a.o_quiz_quiz_answer').each(function () {
                 var $answer = $(this);
                 $answer.find('i.fa').addClass('d-none');
-                if ($answer.find('input[type=radio]')[0].checked) {
-                    if (isCorrect) {
-                        $answer.removeClass('list-group-item-danger').addClass('list-group-item-success');
+                if ($answer.find('input[type=radio]').is(':checked')) {
+                    if (answer.is_correct) {
                         $answer.find('i.fa-check-circle').removeClass('d-none');
+                        $answer.find('span.o_quiz_quiz_answer_text').addClass('text-success');
                     } else {
-                        $answer.removeClass('list-group-item-success').addClass('list-group-item-danger');
-                        $answer.find('i.fa-times-circle').removeClass('d-none');
                         $answer.find('label input').prop('checked', false);
+                        $answer.find('i.fa-times-circle').removeClass('d-none');
+                        $answer.find('span.o_quiz_quiz_answer_text').addClass('text-danger');
+                    }
+                    if (answer.awarded_points > 0) {
+                        var $badge = QWeb.render('quiz.badge', {'answer': answer});
```

Comment:
Looks like this template receives a full answer where it only needs points after all?